aws/resource_aws_elasticsearch_domain.go

```diff
@@ -358,10 +358,10 @@ func resourceAwsElasticSearchDomainUpdate(d *schema.ResourceData, meta interface
 
 	if err := setTagsElasticsearchService(conn, d, d.Id()); err != nil {
 		return err
-	} else {
-		d.SetPartial("tags")
 	}
 
+	d.SetPartial("tags")
+
```

Comment:
> In case where there is an error, function will `return` from function so there is no need to keep `else`

Ah, you're right, good point. Early return FTW 👍